specifyweb/frontend/js_src/lib/components/Forms/SpecifyForm.tsx

```diff
@@ -136,7 +136,9 @@ export function SpecifyForm<SCHEMA extends AnySchema>({
         {formIsLoaded ? (
           <DataEntry.Grid
             aria-hidden={showLoading}
-            className={`${showLoading ? 'pointer-events-none opacity-50' : ''}`}
+            className={`w-full ${
+              showLoading ? 'pointer-events-none opacity-50' : ''
```

Comment:
please format nicely 🙂

```ts
            className={`
               w-full
               ${showLoading ? 'pointer-events-none opacity-50' : ''}
            `}
```


isn't this easier to read and edit?

Reply:
ah, this change might have been made automatically by prettier, in that case sorry about that